content/enterprise_influxdb/v1.8/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,23 @@ menu:
     parent: About the project
 ---
 
+## v1.8.6 [2021-05-21]
+
+{{% note %}}
+**Fine-grained authorization security update.** If you're on InfluxDB 1.8.5, we recommend immediately upgrading to this release. An issue was reported in 1.8.5 where grants with specified permissions for users were not enforced. Versions prior to InfluxDB 1.8.5 are not affected. This security update ensures that only users with sufficient permissions can read and write to a measurement.
+{{% /note %}}
+
+### Features
+
+- **Enhanced Anti-Entropy (AE) logging**: Add an optional `Engine.IsIdle()` trace logging to support debugging improvements in the Anti-Entropy service. Add an `IsLogged` parameter to `tsdb.Engine.IsIdle`. When trace logging is turned on, this option reports the reasons why a shard is not idle.
```

Comment:
Does it make sense to list the new reasons what a shard is not idle in this section:
https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/anti-entropy/#anti-entropy-log-messages

Reply:
Maybe helpful at some point to fill this log message content in a bit more? I can create an issue if it makes sense to do so.

Reply:
Only if we also note that this requires `Debug` level logging, and is given per hot shard.
```
"not idle because of active Cache compactions",
"not idle because of active Level Zero compactions",
"not idle because of active Level One compactions",
"not idle because of active Level Two compactions",
"not idle because of active Full compactions",
"not idle because of active TSM Optimization compactions"
"not idle because cache size is nonzero"
"not idle because shard is not fully compacted"
```

Reply:
But, this only will show up in Enterprise when AE is used.  So, not an OSS user-visible feature, just implemented there of necessity.

Reply:
Thank you, @davidby-influx - would folks change this setting [here](https://docs.influxdata.com/enterprise_influxdb/v1.8/administration/config-data-nodes/#level--info) to from level ="info" to level="debug"?

We can certainly add this info, but doesn't appear we have a task-based doc for how to do this.

Reply:
Yup, that's the spot.

Reply:
I don't think end-users of Enterprise should run with debug level logging until and unless our support team requests it for assistance with a specific problem.  Too verbose, otherwise.

Reply:
The statement above in the change log is erroneous, due to my stupidity.  One need not turn on trace logging to get the new AE information, only set the general logging level to `debug`.  The trace logging was the original design, which morphed during the implementation.